class/defaults.yml

```
@@ -55,6 +55,8 @@ parameters:
         enabled: ${cilium:egress_gateway:enabled}
       bpf:
         masquerade: true
+      l2announcements:
+        enabled: ${cilium:l2_announcements:enabled}
```

I'd probably configure increased K8s client rate limits in the component defaults since this will easily get forgotten in the future. In the defaults, I'd probably set limits that should support at least 20 services (which would need 10QPS according to the formula in the docs), but the nodes need some extra QPS for other controllers, so maybe something like:

```suggestion
        enabled: ${cilium:l2_announcements:enabled}  
      k8sClientRateLimit:
        qps: 15
        burst: 30
```